tests/queries/0_stateless/02021_create_database_with_comment.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+DB_NAME="${CLICKHOUSE_DATABASE}"
```

Comment:
There was a PR that allows to access database name from `sql` test, but it wasn't merged https://github.com/ClickHouse/ClickHouse/pull/27107/files , because there's solution for `ReplicatedMergeTree`

Reply:
I do not think that this would simplify tests in any way